packages/math/init.lua

```diff
@@ -42,9 +42,13 @@ function package.declareSettings (_)
    })
    SILE.settings:declare({
       parameter = "math.font.size",
-      type = "integer",
-      default = 10,
+      type = "number or integer",
```

Comment:
I think we should consider whether we ought to allow relative values here such that formulas would track changes to the document font size in a relative way.